app/service/redactor_notification_service.rb

```diff
@@ -0,0 +1,17 @@
+class RedactorNotificationService
```

Comment:
maybe you can put all the notifications in a folder? to make it easier for other users to use the services folder?